### PR TITLE
update android v + make changelog meaningful

### PR DESCRIPTION
**Pull request by BrodyHughes**
Fixes APP-####

### CODE REVIEW

**Review by ibrahimtaveras00 — APPROVED** [review]

**Review by jkadamczyk — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,10 +17,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Changed
 
-- hotfix
```
Comment (skylarbarrera) [inline CHANGELOG.md:20]: smh

**Review by skylarbarrera — APPROVED** [review]
:shipit:

File: CHANGELOG.md (hunk 1 above)
Comment (BrodyHughes) [reply CHANGELOG.md:20]: lmao

Comment (BrodyHughes) [reply CHANGELOG.md:20]: tbf it made the v bump a little early so wasn't 100% sure what all was going to be in it 😅

**Pull request merged**